.github/workflows/test_fast.yml

```
@@ -77,8 +83,10 @@ jobs:
           bandit -r --ini .bandit cylc/flow
 
       - name: Integration Tests
+        timeout-minutes: 6
         run: |
-          pytest tests/integration
+          pytest -n 0 --dist=no tests/integration
+        # Turned off x-dist due to flakiness on GH Actions for some reason
```

Would rather not change something just to cover up issues at our end, at least until we understand what the issues are?

Can we at least try with just the other fixes first and see whether this is necessary after?

```suggestion
          pytest tests/integration
```

---

Removed this change